designer/client/form-details.js

```diff
@@ -51,7 +52,7 @@ class FormDetails extends React.Component {
   }
 
   handleIsFeedbackFormRadio = (e) => {
-    const isFeedbackForm = e.target.value;
+    const isFeedbackForm = str2bool(e.target.value);
```

Comment:
`isFeedbackForm = e.target.value === "true"` was nicer imo. Since there are only two options true or false, you only have to evaluate once. the str2bool function is only helpful if you are going to be casting a bunch of string values.

Reply:
we are passing boolean values to Radios, not sure why we are getting string values, so this function is just catering for future changes, if the Radios pass back boolean values